src/locales/es/duration.js

```diff
@@ -0,0 +1,32 @@
+// A. Either re-use phrases from iso-duration...
```

Comment:
Maybe delete the comments in this file, they are not really needed.

Reply:
Sure.